apis/core/test/domain/cube-projects/create.test.ts

```diff
@@ -209,5 +209,95 @@ describe('domain/cube-projects/create', () => {
         }],
       })
     })
+
+    it('creates a properly defined sources collection', async () => {
+      // given
+      const resource = clownface({ dataset: $rdf.dataset() })
+        .namedNode('')
+        .addOut(rdfs.label, 'Foo bar project')
+        .addOut(cc.projectSourceKind, shape('cube-project/create#CSV'))
+
+      // when
+      const project = await createProject({ resource, store, projectsCollection, user })
+      const mapping = await store.get(project.csvMapping?.id as NamedNode)
+      const sourceCollection = await store.get(mapping?.out(cc.csvSourceCollection).term as NamedNode)
+
+      // then
+      expect(sourceCollection).to.matchShape({
+        property: {
+          path: hydra.manages,
+          xone: [{
+            node: {
+              property: [{
+                path: hydra.property,
+                hasValue: rdf.type,
+                minCount: 1,
+              }, {
+                path: hydra.object,
+                hasValue: cc.CSVSource,
+                minCount: 1,
+              }],
+            },
+          }, {
+            node: {
+              property: [{
+                path: hydra.property,
+                hasValue: cc.csvMapping,
+                minCount: 1,
+              }, {
+                path: hydra.object,
+                hasValue: mapping!.term,
+                minCount: 1,
+              }],
+            },
+          }],
```

Comment:
Using `sh:xone` ensures that every value of `hydra:manages` satisfies at least one of the possible shapes